rich/jupyter.py

```diff
@@ -14,6 +14,12 @@
 <pre style="white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace">{code}</pre>
 """
 
+JUPYTER_CLASSES_TO_NOT_RENDER = {
```

Comment:
AFAICT this is only used in a single context. It doesn't need to be publicly exposed so we could move it to the file where it is referenced.